pkg/commands/oscommands/os.go

```diff
@@ -301,10 +301,15 @@ func sanitisedCommandOutput(output []byte, err error) (string, error) {
 // OpenFile opens a file with the given
 func (c *OSCommand) OpenFile(filename string) error {
 	commandTemplate := c.Config.GetUserConfig().OS.OpenCommand
+	quoted := c.Quote(filename)
+	if c.Platform.OS == "linux" {
```

Comment:
I feel like this logic should live inside the `Quote` function itself.

Reply:
Agreed! The issue though is that we only really need the extra logic for the open file command specifically.
Adding the extra quoting for all cases on Linux would brake other commands :(

Reply:
ah damn